src/ast.cpp

```diff
@@ -2226,4 +2226,20 @@ namespace Sass {
     }
   }
 
+  //////////////////////////////////////////////////////////////////////////////////////////
+  // Convert map to (key, value) list.
+  //////////////////////////////////////////////////////////////////////////////////////////
+  List* Map::to_list(Context& ctx, ParserState& pstate) {
```

Comment:
> In this case it might be nicer to add a List constructor that takes a Map argument.
 Add a line note

This runs into the issue of not being able to create new list nodes in AST nodes.